content/v2.0/reference/internals/storage-engine.md

```diff
@@ -0,0 +1,106 @@
+---
+title: InfluxDB storage engine
+description: >
+  An overview of the InfluxDB storage engine architecture.
+weight: 7
+menu:
+  v2_0_ref:
+    name: Storage engine
+    parent: InfluxDB Internals
+v2.0/tags: [storage, internals]
+---
+
+The InfluxDB storage engine ensures that
+
+- Data is safely written to disk
+- Queried data is returned complete and correct
+- Data is accurate (first) and performant (second)
+
+This document outlines the internal workings of the storage engine.
+This information is presented both as a reference and to aid those looking to maximize performance.
```

Comment:
Learn how the storage engine works and how to maximize performance.

Reply:
Idea: keep the descriptive language, but use this imperative in the description header.

Reply:
```
description: >
  Learn how the storage engine works and how to maximize performance.
```